Automattic/wc-calypso-bridge
# disable WC Admin dev docs menu item by default

See #462 

This PR adds 

- a class for WooCommerce Admin customizations
- removes the `Dev Docs` menu item by default since people using Calypso are unlikely to be doing dev work in that environment
- Dev Docs can be restored with the `wc_calypso_bridge_development_mode` filter

### Testing

- Switch WC Admin to https://github.com/woocommerce/woocommerce-admin/tree/add/feature-config-filter
- Add the following filter
```
add_filter( 'wc_calypso_bridge_development_mode', '__return_false', 1001 );
```
- The Dev Docs menu item should be removed.
